specs/APIReview_TextureStream.md

```diff
@@ -0,0 +1,353 @@
+	
+# Background
+Many native apps use a native engine for real-time communication scenarios, which include video 
+capture, networking and video rendering.  However, often, these apps still use WebView or 
+Electrion for UI rendering. The separation between real-time video rendering and UI rendering 
+prevents apps from rendering real-time video inside the web contents. This forces apps to
+render the real-time video on top of the web contents, which is limiting. Rendering video on 
+top constrains the user experience and it may also cause performance problems.  
+We can ask the native apps to use web renderer for video handling because web standard already 
+provides these features through WebRTC APIs. The end developers, however, prefer to use 
+their existing engine such as capturing and composition, meanwhile using WebRTC API for rendering.
+
+# Description
+The proposed APIs will allow the end developers to stream the captured or composed video frame to 
+the WebView renderer where Javascript is able to insert the frame to the page through W3C standard 
+API of Video, MediaStream element for displaying it.
+The API will use the shared GPU texture buffer so that it can minimize the overall cost with 
+regards to frame copy.
+
+# Examples
+Javascript
+// User click the video capture button.
+document.querySelector('#showVideo').addEventListener('click',
+  e => getStreamFromTheHost(e));
+async function getStreamFromTheHost(e) {
+  try {
+    // Request stream to the host with unique stream id.
+    const stream = await window.chrome.webview.getTextureStream('webview2-abcd1234'); 
```

Comment:
This sample doesn't demonstrate JS handling stop and restarting (is it via active/inactive events?). Is that expected or does it not explicitly handle that?